### DISCUSSION TITLE
Non paywalled version without the ability to log in

### DISCUSSION THREAD

**Discussion by s-liwka** [Docile-Alligator/Infinity-For-Reddit#1589] [Ideas]
I am using Infinity without an account and it's working great. Because it I assume this use case wasn't affected by the API change and I don't see why the developer shouldn't maintain a version that doesnt let you log in for users who dont need that

**Comment by iTrooz** [comment]
+1

To add more details:
The anonymous API still works (although rate limited), so for now I am using Infinity version 6.1.1 (last version without paywall)
But I'd like to upgrade to the latest version to get the newest features. Would it be possible to add a X button to close the paywall (and not show it again) so that we can still use the anonymous Reddit API ?
